.gitignore

```diff
@@ -4,6 +4,7 @@ integration/examples/bazel/bazel-*
 integration/examples/test-plugin/local/bazel/bazel-*
 *.new
 .idea/
+.vscode/
```

Comment:
Is that related to your PR?

Reply:
Sort of, I'm working in vscode and didn't want to commit the .vscode folder by accident. Should this be split into a different PR?